### ISSUE TITLE
Should intentional choice of administrative set be required, when more than one choice is possible?

### ISSUE THREAD

**Issue by newmanld** [samvera/hyrax#678]
### Descriptive summary

On a work submission form (software version as demo'd by @jcoyne at LDCX, workflow whiteboard breakout group, on 3/29/17), on the relationships tab where a drop-down list of administrative sets now appears, the depositor is not required to make a deliberate choice, when more than one administrative set is possible for that depositor.  Instead the application will select the first administrative set in the list for the depositor, if the depositor does not intentionally select a value.    

### Rationale

As a manager/administrator who has authorized depositors for more than one administrative set, I want those depositors to deliberately choose the set, so that the correct workflow is applied.  

Conversely, as a manager/administrator with some (or all) depositors authorized for only one administrative set, I do not want those depositors to be confused by the choice of administrative set when the drop-down list of administrative sets has only one value.  This should perhaps be written up as a separate issue.

### Expected behavior
When more than one choice of administrative set is possible, a new submission is not complete until the depositor has selected an administrative set.

**Comment by newmanld** [comment]
@jeremyf tagging you at your request as a reminder about the documenting group implementations concern.

**Comment by jcoyne** [comment]
@ggeisler your thoughts?

**Comment by ggeisler** [comment]
I think there are two possible approaches and choosing the right one depends on what deposit scenario we want to favor. I'll present the two approaches and ask @hannahfrost to weigh in with her repository manager/administrator and product manager point of view.

Scenario 1: The repository has many occasional end-user depositors who don't really understand, remember, or care what an admin set is. They just want to get their work submitted. For this scenario, showing an admin set choice preselected in the select menu, as long as it is the admin set that is the "default" or most general admin set (e.g., an admin set intended for any institutional user to self-deposit), is desirable because it streamlines the deposit process and doesn't force the user to take explicit action to select what would otherwise be the default choice.

Scenario 2: As @newmanld brings up, however, there are at least some scenarios where the repository manager really wants the user to deliberately select an admin set. So the solution best for scenario 1 above would make it too easy for a user to leave the default value and not consciously select a more appropriate set for the work. The best solution for this scenario is to not show an initial choice in the admin set select menu. We'd just show "Select an administrative set..." and force the user to select a choice from the menu.

Basically it is a tradeoff between making it easier for users who will use the default admin set but making it more likely some users will end up using the default set when they should've made a more appropriate choice for their work, and guarding against that danger but making all users to take the extra step of explicitly choosing an admin set. What do you think, @hannahfrost?

**Comment by vantuyls** [comment]
Would it be possible to force the AdminSet picker to a default AdminSet *by* WorkType? That is, could we define the behavior of the AdminSet picker based on the WorkType we're depositing, allowing for flexibility to run Scenario1 or Scenario2?

**Comment by newmanld** [comment]
@vantuyls see #674?

**Comment by hannahfrost** [comment]
@ggeisler @newmanld  I favor Scenario 2 where the depositor makes an explicit choice of admin set when more than one admin set (the default one) exists.

**Comment by jeremyf** [comment]
@newmanld I believe you summed things up well (and I'm forgetting the group implementation concern). Though it may be summed up in #674.

We want to minimize user decisions. First we ask what they are depositing (e.g. Document, Video), then we filter their Admin Set options based on 1) Can I create a work in this Admin Set and 2) Does this Admin Set allow this work type to be created.

**Comment by rjkati** [comment]
In Hyrax 3.0.2 on nurax-dev, both the "Generic Work" and "Image" forms select the Default Admin Set by default, although users can choose another admin set from the list. Default Admin Set appears at the bottom of the list. @jlhardes

**Comment by jlhardes** [comment]
We haven't had any further comments or discussion about this since 2017 and it is possible to choose a different admin set even though there is a default selection of the Default Admin Set. I am inclined to close this issue for now since there isn't anything broken about how this is working and there is no current call to change this process.
